com.vivaldi.Vivaldi.yaml

```diff
@@ -0,0 +1,124 @@
+  - --filesystem=home/.local/share/applications:create # Allow for PWAs
+  - --filesystem=home/.local/share/icons:create # Allow for PWAs
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-download # Default download directory
+  - --filesystem=xdg-run/dconf # For GNOME proxy resolution
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=~/.config/dconf:ro # For GNOME proxy resolution
+  - --filesystem=~/.config/kioslaverc # For KDE proxy resolution (KDE5 only)
+  - --filesystem=~/.local/lib:create # Extra FFMpeg saved here
+  - --own-name=org.mpris.MediaPlayer2.chromium.* # Like the Opera flatpack (on flathub) this is still hardcoded to Chromium :(
+  - --persist=.pki
+  - --require-version=1.8.2
+  - --share=ipc
+  - --share=network
+  - --socket=cups # Printing
+  - --socket=pcsc # Webauthn / FIDO
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --system-talk-name=org.freedesktop.Avahi
+  - --system-talk-name=org.freedesktop.UPower
+  - --talk-name=ca.desrt.dconf # For GNOME proxy resolution
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.freedesktop.FileManager1
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.kde.kwalletd5 # Used for password encryption under KDE environments
+
+modules:
+  - name: dconf
+    buildsystem: meson
+    config-opts:
+      - -Dbash_completion=false
+      - -Dman=false
+      - -Dvapi=false
+    cleanup:
+      - /etc
+      - /include
+      - ca.desrt.dconf.service
+      - dconf.service
+      - dconf-service
+      - '*.pc'
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/dconf/0.40/dconf-0.40.0.tar.xz
+        sha256: cf7f22a4c9200421d8d3325c5c1b8b93a36843650c9f95d6451e20f0bcb24533
+        x-checker-data:
+          type: gnome
+          name: dconf
+          stable-only: true
+      # From: https://src.fedoraproject.org/rpms/dconf
+      - type: patch
+        path: dconf-override.patch
+
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        tag: v2022.04
+        commit: 55a60d110301e8dd37b631503c3524ab7baaf7aa
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+
+  - name: vivaldi
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/vivaldi
+      - bsdtar xf vivaldi.deb
+      - bsdtar -C /app/vivaldi --strip-components 3 -xf data.tar.xz ./opt/vivaldi ./usr/share/applications/vivaldi-stable.desktop
+      - sed -i 's/LD_PRELOAD/ZYPAK_LD_PRELOAD/g;/CHROME_VERSION_EXTRA/i export VIVALDI_FFMPEG_FOUND=YES \# Prevents excessive warning on first start for flatpak users' /app/vivaldi/vivaldi
+      - sed -i 's,^Exec=.*vivaldi-stable,Exec=/app/bin/vivaldi,;s,^Icon=vivaldi,Icon=com.vivaldi.Vivaldi,' /app/vivaldi/applications/vivaldi-stable.desktop
```

Comment:
Drop `/app/bin/vivaldi` just use `vivaldi`.

Desktop-file-edit worn't work here because the file has multiple actions that desktop-file-edit can't change, it can only change keys in the main entry